themes/stanford_basic/src/scss/components/spacer/_spacer.scss

```diff
@@ -0,0 +1,21 @@
+@charset "UTF-8";
```

Comment:
Improper tab spacing. Tabs should be 2 characters

Reply:
fixed.